packages/config/tailwind-preset.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   content: [
     "./pages/**/*.{js,ts,jsx,tsx}",
     "./components/**/*.{js,ts,jsx,tsx}",
-    "../../packages/app-store/**/{components,pages}/**/*.{js,ts,jsx,tsx}",
+    "../../packages/app-store/**/*{components,pages}/**/*.{js,ts,jsx,tsx}",
```

Comment:
Correcting the glob to take into account "_components" and "_pages" within app-store and "components" and "pages" within app folders.